ui/components/src/components-react/filtering/FilteringInput.tsx

```diff
@@ -43,7 +43,7 @@ export interface FilteringInputProps extends CommonProps {
   onFilterClear: () => void;
   /**
    * Tells the component what is the status of filtering.
-   * @beta
+   * @public
```

Comment:
I think this should be removed instead of making it `public` as whole interface is marked as `public`. 

There are other places too where `beta` tag on properties of `public` interface are changed instead of removed.

Reply:
Let me know if you see any tags that I missed on interface members.